jerry-core/lit/lit-char-helpers.cpp

```diff
@@ -112,6 +114,10 @@ lit_char_is_unicode_letter (ecma_char_t c) /**< code unit */
   {
     return true;
   }
+  else if (c <= LIT_UTF8_1_BYTE_CODE_POINT_MAX)
+  {
+    return false;
```

Comment:
Could you, please, add a comment like `/* Other one-byte code points are not corresponding to letters */`?